### PR TITLE
Deprecate evaluation from the about menu

### PR DESCRIPTION
**Pull request by hannahbergam**
Deprecates instances of /evaluation and adds redirects for that and all subpages.

Before/After

Screenshot 2024-08-20 at 11 15 01 AM
Screenshot 2024-08-20 at 11 14 52 AM

## Links
- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-2260

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/research.md [hunk 1]
```diff
@@ -41,13 +41,6 @@ separately.
 * Findings show that after completing one Hour of Code activity students report liking computer science more, feel that they are better able to learn computer science, and are better at computer science than their peers.
 
 
-Other reports
-
-- [Code.org Evaluation Report 2015-16 (PDF)](/files/EvaluationReport2015-16.pdf)
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/research.md:46]: Just to confirm we want to remove this? Should `/files/EvaluationReport2015-16.pdf` also be removed?

**Review by TurnerRiley — APPROVED** [review]
LGTM! Just had one quick question!

File: pegasus/sites.v3/code.org/public/research.md (hunk 1 above)
Comment (hannahbergam) [reply pegasus/sites.v3/code.org/public/research.md:46]: Luckily, it sounds like Dani is hoping to delete the section but keep the PDF (linked from an annual report) https://codedotorg.slack.com/archives/C0453TUMLE7/p1724190729875179. Thanks for checking!

Comment (TurnerRiley) [reply pegasus/sites.v3/code.org/public/research.md:46]: Sounds good!

**Pull request merged**